models/classes/LtiProvider/LtiProvider.php

```diff
@@ -92,7 +92,7 @@ public function __construct(
         string $toolAudience = null,
         string $toolOidcLoginInitiationUrl = null,
         string $toolLaunchUrl = null,
-        string $toolPublicKey = null,
+        string $toolPublicKey = '',
```

Comment:
Why  do you prefer `toolPublicKey` to be an empty string, but `toolJwksUrl` to be null for empty values? Why it's so different?

Reply:
agreed 👍